packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -153,7 +153,7 @@ export const Table: FC<Props> = memo((props: Props) => {
       stateReducer: stateReducer,
       initialState: getInitialState(props, memoizedColumns),
     }),
-    [memoizedColumns, memoizedData, stateReducer, resizable]
+    [memoizedColumns, memoizedData, resizable, stateReducer, props]
```

Comment:
nit: Best practices is to destruct nested props and use them in the dependencies.

Reply:
done